.travis.yml

```diff
@@ -108,15 +108,15 @@ matrix:
     compiler: gcc
     env:
     - Q_OR_C_MAKE=cmake
-    - QT_VERSION=56 # actually Qt 5.6, used to check minimum supported Qt works
+    - QT_VERSION=57 # actually Qt 5.6, used to check minimum supported Qt works
```

Comment:
Perhaps the comment needs revising to say 5.7 (it is explaining the use of the string containing the integer "57" - of course once we get above "59"/ Qt 5.9 we might have to reconsider what we use - but it is only a label so it can be anything, not just "57"!)

Reply:
Updated!